### PR TITLE
bump to 2.19.0

### PR DESCRIPTION
**Pull request by jshilong**
## Motivation

Bump mmdet to 2.19.0

## Modification

Update the changelog and all related version

### CODE REVIEW

File: README_zh-CN.md [hunk 1]
```diff
@@ -59,10 +59,10 @@ MMDetection 是一个基于 PyTorch 的目标检测开源工具箱。它是 [Ope
 
 ## 更新日志
 
-最新的 **2.18.1** 版本已经在2021.11.15发布:
-- 发布了 [QueryInst](http://arxiv.org/abs/2105.01928) 的预训练模型。
-- 支持绘制混淆矩阵。
-- 修复了 PVT 中的 SpatialReductionAttention 以及 PVT 和 Swin-Transformer 中 trunc_normal_init 的错误用法。
+最新的 **2.19.0** 版本已经在2021.11.29发布:
```
Comment (hhaAndroid) [inline README_zh-CN.md:62]: 在 2021.11.29 发布:

File: README_zh-CN.md [hunk 2]
```diff
@@ -59,10 +59,10 @@ MMDetection 是一个基于 PyTorch 的目标检测开源工具箱。它是 [Ope
...
+- 支持了 [Label Assignment Distillation](https://arxiv.org/abs/2108.10520) (#6342)
```
Comment (hhaAndroid) [inline README_zh-CN.md:63]: No need to write PR ID

File: README.md [hunk 3]
```diff
@@ -60,10 +60,11 @@ This project is released under the [Apache 2.0 license](LICENSE).
 
 ## Changelog
 
-**2.18.1** was released in 15/11/2021:
-- Release [QueryInst](http://arxiv.org/abs/2105.01928) pre-trained weights.
-- Support plot confusion matrix.
-- Fix SpatialReductionAttention in PVT and fix trunc_normal_init in both PVT and Swin-Transformer.
+**2.19.0** was released in 29/11/2021:
+
+- Support [Label Assignment Distillation](https://arxiv.org/abs/2108.10520) (#6342)
```
Comment (ZwwWayne) [inline README.md:65]: PR ID is unnecessary

File: docs/changelog.md [hunk 4]
```diff
@@ -1,5 +1,46 @@
 ## Changelog
 
+### v2.19.0 (29/11/2021)
+
+#### Highlights
+
+- Support [Label Assignment Distillation](https://arxiv.org/abs/2108.10520) (#6342)
```
Comment (ZwwWayne) [inline docs/changelog.md:7]: PR ID is unnecessary in highlights

File: docs/changelog.md [hunk 5]
```diff
@@ -1,5 +1,46 @@
...
+
+
+#### Documents
+
+- Update README.md (#6567)
+
+#### Contributors
+
+A total of 11 developers contributed to this release.
+Thanks @FloydHsiu, @RangiLyu, @ZwwWayne, @AndreaPi, @st9007a, @hachreak, @BIGWangYuDong, @hhaAndroid, @AronLin, @chhluo, @vealocia, @@HarborYuan, @st9007a
```
Comment (ZwwWayne) [inline docs/changelog.md:38]: add jshilong

**Comment by ZwwWayne** [comment]
Also change this line to 1.5.0 https://github.com/open-mmlab/mmdetection/blob/master/mmdet/__init__.py#L20

File: docs/changelog.md (hunk 5 above)
Comment (RangiLyu) [inline docs/changelog.md]:
```suggestion
Thanks @FloydHsiu, @RangiLyu, @ZwwWayne, @AndreaPi, @st9007a, @hachreak, @BIGWangYuDong, @hhaAndroid, @AronLin, @chhluo, @vealocia, @HarborYuan, @st9007a
```

File: mmdet/__init__.py [hunk 6]
```diff
@@ -17,7 +17,7 @@ def digit_version(version_str):
 
 
 mmcv_minimum_version = '1.3.8'
-mmcv_maximum_version = '1.4.0'
+mmcv_maximum_version = '1.5.0'
```
Comment (hhaAndroid) [inline mmdet/__init__.py:20]: Why modify this？

Comment (jshilong) [reply mmdet/__init__.py:20]: @ZwwWayne

**Pull request merged**